apps/classroomio-com/static/manifest.json

```diff
@@ -1,25 +1,15 @@
 {
-	"background_color": "#ffffff",
-	"theme_color": "#333333",
-	"name": "ClassroomIO",
-	"short_name": "ClassroomIO",
-	"display": "minimal-ui",
-	"start_url": "/",
-	"icons": [
-		{
-			"src": "logo-32.png",
-			"sizes": "32x32",
-			"type": "image/png"
-		},
-		{
-			"src": "logo-192.png",
-			"sizes": "192x192",
-			"type": "image/png"
-		},
-		{
-			"src": "logo-512.png",
-			"sizes": "512x512",
-			"type": "image/png"
-		}
-	]
```

Comment:
@Chifez You shouldn't remove these icons.

You need to make that you download the right resolution from the figma and add that into the repo.

Please make sure to keep also the file name so we can use it when we need to select large images or smaller images. Also look into the static folder for all `logo-*` those are the images you should update not the code.

In addition please we need to have different types of images like the one with white background, blue background and transparent background. We will use it for different purposes.